src/app/components/AllowanceMenu/index.tsx

```diff
@@ -145,26 +202,69 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
                 onChange={(e) => setBudget(e.target.value)}
               />
             </div>
-            <Setting
-              title={t("enable_login.title")}
-              subtitle={t("enable_login.subtitle")}
-            >
-              <Toggle
-                checked={lnurlAuth}
-                onChange={() => setLnurlAuth(!lnurlAuth)}
-              />
-            </Setting>
+            <div className="pb-4 border-b border-gray-200 dark:border-neutral-500">
+              <Setting
+                title={t("enable_login.title")}
+                subtitle={t("enable_login.subtitle")}
+              >
+                <Toggle
+                  checked={lnurlAuth}
+                  onChange={() => setLnurlAuth(!lnurlAuth)}
+                />
+              </Setting>
+            </div>
+            <h2 className="mt-4 text-lg text-gray-900 font-bold dark:text-white">
+              {t("edit_permissions")}
+            </h2>
+
+            {isLoadingPermissions && (
+              <div className="flex justify-center">
+                <Loading />
+              </div>
+            )}
+
+            {hasPermissions && (
+              <div>
+                {permissions.map((permission, index) => (
+                  <div key={index} className="flex items-center my-2">
+                    <Checkbox
+                      id={`permission-${permission.id}`}
+                      name={`permission-${permission.id}`}
+                      checked={permission.enabled}
+                      onChange={() => {
+                        setPermissions(
+                          permissions.map((prm) =>
+                            prm.id === permission.id
+                              ? { ...prm, enabled: !prm.enabled }
+                              : prm
+                          )
+                        );
+                      }}
+                    />
+                    <label
+                      htmlFor={`permission-${permission.id}`}
+                      className="ml-2 block text-sm text-gray-900 font-medium dark:text-white"
+                    >
+                      {permission.method}
+                    </label>
+                  </div>
+                ))}
+              </div>
+            )}
+
+            {isEmptyPermissions && (
+              <p className="text-gray-500 dark:text-neutral-400">
+                {t("no_permissions")}
```

Comment:
we just don't show anything if there are no permissions stored. 
most websites will initially have no such specific permissions.

Reply:
Agree, no need to show a default on empty permissions (yet)